src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/EditAndSyncFeatures/README.md

```diff
@@ -0,0 +1,14 @@
+# Edit and sync features
+
+This sample demonstrates how to synchronize offline edits with a feature service.
+
+<img src="EditAndSyncFeatures.jpg" width="350"/>
+
+### Instructions
+
+1. Pan and zoom to the area you would like to download features for, ensuring that all features are within the rectangle.
+2. Tap the 'generate' button. This will start the process of generating the offline geodatabase.
+3. Tap on a feature within the area of the generated geodatabase. Then tap on the screen (anywhere within the range of the local geodatabase) to move the feature to that location.
```

Comment:
Explicitly tell the user in the instructions that they need to tap on a "Point" feature to move. They cannot click and move polylines and polygons!